### PR TITLE
Control minimum (maximum) allowable value

### PR DESCRIPTION
**Pull request by ishchegolev**
new tags:
- `envMinValue`
- `envMaxValue`

### CODE REVIEW

**Comment by medyagh** [comment]
I like the idea ! @ishchegolev could you please make sure it passes the tests?

File: env_test.go [hunk 1]
```diff
@@ -571,6 +571,82 @@ func TestTextUnmarshalerError(t *testing.T) {
 	assert.Error(t, env.Parse(cfg))
 }
 
+func TestMinMaxBorders(t *testing.T) {
```
Comment (medyagh) [inline env_test.go]: could you please add a test to mix it with other ones ? like envExpand ?

**Comment by medyagh** [comment]
@ishchegolev for example add a test for see how it works with envExpand , if a variable is being exandped will it still follow the min max restrictions ?

**Comment by caarlos0** [comment]
Hi @ishchegolev !

The goal of the lib is parsing config from Env into a struct. I think that validating the config is out of that scope.

There are libs like https://github.com/go-playground/validator that do that already...

File: env_test.go (hunk 1 above)
Comment (ishchegolev) [reply env_test.go]: done in 01fb3a8309094cd594c7f7d2ec8483f13a6059eb

**Comment by caarlos0** [comment]
giving my last comment (https://github.com/caarlos0/env/pull/60#issuecomment-426026168), closing this.

Sorry for the time you invested on it, but I rather keep the library simple.

**Pull request closed without merging**